install/kubernetes/cilium/values.yaml

```diff
@@ -12,6 +12,10 @@ agent:
 config:
   enabled: true
 
+# Do not include the hubble-cli DaemonSet by default
```

Comment:
this could possibly move under the `hubble` section so that all the hubble-related settings are in a single section:

```
hubble:
  cli:
    enabled: false
```

Reply:
Great idea! I have added a commit to accommodate this.